testdata/p4_16_samples/issue4796.p4

```diff
@@ -0,0 +1,56 @@
+#include <core.p4>
```

Comment:
this could be probably smaller to highlight just the problem.
you don't need to use pna.p4

Reply:
Yeah, right. Just wanted to add exactly the testcase from the issue.